packages/react/src/patterns/blocks/ContentBlockMedia/ContentBlockMediaItem.js

```diff
@@ -16,27 +16,29 @@ const { stablePrefix } = ddsSettings;
 const { prefix } = settings;
 
 /**
- * UsecasesItem
+ * ContentBlockMediaItem
  *
  * @param {object} props props object { lists:{ title, copy, link: { text, href, target}} }
- * @param {string} props.lists.title UsecasesItem title property
- * @param {string} props.lists.copy UsecasesItem copy property
- * @param {object} props.lists.link UsecasesItem link object
- * @param {string} props.lists.link.href UsecasesItem href property of link object
- * @param {string} props.lists.link.title UsecasesItem text property of link object
- * @param {string} props.lists.link.target UsecasesItem target property of link object
+ * @param {string} props.lists.title ContentBlockMediaItem title property
+ * @param {string} props.lists.copy ContentBlockMediaItem copy property
+ * @param {object} props.lists.link ContentBlockMediaItem link object
+ * @param {string} props.lists.link.href ContentBlockMediaItem href property of link object
+ * @param {string} props.lists.link.title ContentBlockMediaItem text property of link object
+ * @param {string} props.lists.link.target ContentBlockMediaItem target property of link object
  * @returns {object} JSX Object
  */
-const UsecasesItem = ({ lists: { title, copy, link } }) => {
+const ContentBlockMediaItem = ({ lists: { title, copy, link } }) => {
```

Comment:
I think this is what is in the `ContentGroupSimple`, so if we can get it to use that this might no longer be needed.

Reply:
Cool, will work it out with @RaphaelAmadeu and see where he is with it and how to incorporate. Might need to be a separate story for another sprint if he's currently working on it.